util/src/main/java/com/google/cloud/hadoop/util/AsyncWriteChannelOptions.java

```diff
@@ -47,12 +54,19 @@ public enum UploadType {
     /* Write whole file to disk and then upload.*/
     WRITE_TO_DISK_THEN_UPLOAD,
     /* Write chunks to file along with uploading to gcs, and failure will be retried from data on disk.*/
-    JOURNALING
+    JOURNALING,
+    /* Write are performed using parallel composite upload strategy.  */
+    PARALLEL_COMPOSITE_UPLOAD
   }
 
+  private static final int PARALLEL_COMPOSITE_UPLOAD_BUFFER_COUNT = 1;
```

Comment:
why 1? A comment will be helpful how we came up with this magic number.

Reply:
added.